### DISCUSSION TITLE
請問拼音是默認預設不能關閉的嗎？

### DISCUSSION THREAD

**Discussion by longsang0688** [rime/home#1738] [Q&A]
你好，我正在使用鼠鬚管的五筆畫，發現使用筆畫時也會同時使用拼音。
我的筆畫只有UIOJK  代表橫、豎、撇、捺、折
當我按下「O」時就會先顯示拼音的「哦」字
按下「A」就會顯示顯示拼音的「啊」字
其他字母也是一樣

所以我估計拼音也是同時運作的
雖然我已刪除了luna_pinyin有關的字眼和刪除yaml文件
只剩下stroke

可是拼音還是會出現
這是因為拼音是無法刪除的嗎？？

謝謝！

**Comment by lotem** [comment]
預設的筆畫輸入方案開了拼音反查：

https://github.com/rime/rime-stroke/blob/65fdbbf2f9485cc907cb9a6d6b9210938a50f71e/stroke.schema.yaml#L45
https://github.com/rime/rime-stroke/blob/65fdbbf2f9485cc907cb9a6d6b9210938a50f71e/stroke.schema.yaml#L70

所以會出拼音。至於你的方案如何定義。那就不知道了。沒有任何欽定的拼音。方案要用纔會用。

**Reply by longsang0688** [reply]
原來如此，謝謝你！
